examples/starknet_hello_world/src/lib.cairo

```diff
@@ -1,58 +1,65 @@
-#[abi]
-trait IBalance {
-    fn increase_balance(amount: felt252);
-    fn get_balance() -> felt252;
+#[starknet::interface]
+trait IBalance<T> {
+    fn replace(ref self: T, a: u128) -> u128;
 }
 
-#[contract]
+#[starknet::contract]
 mod Balance {
+    use traits::Into;
+
+    #[storage]
     struct Storage {
-        balance: felt252, 
+        value: u128, 
     }
 
     #[constructor]
-    fn constructor(balance_: felt252) {
-        balance::write(balance_);
+    fn constructor(ref self: ContractState, value_: u128) {
+        self.value.write(value_);
     }
 
-    // Increases the balance by the given amount.
     #[external]
```

Comment:
The blogpost https://community.starknet.io/t/cairo-1-contract-syntax-is-evolving/94794 had something about external now changing to `#[external(v0)]`. Shouldn't this be changed here?